src/service/arguments/action/action-arguments.ts

```diff
@@ -0,0 +1,93 @@
+import { FlowType, InputValues, LoggerLevel } from "@bc/domain/inputs";
+import * as core from "@actions/core";
+import { OptionValues } from "commander";
+import Container from "typedi";
+import { ParsedInputs } from "@bc/service/inputs/parsed-inputs";
+import { InvalidInput } from "@bc/domain/errors";
+
+/**
+ * Parses all inputs from github action workflow files
+ */
+export class ActionArguments {
+    /**
+     * Converts user input to corresponding FlowType enum
+     * @param flowType value for "flow-type" input
+     * @returns corresponding enum
+     */
+    private getFlowType(flowType: string): FlowType {
+        switch (flowType) {
+            case "pull-request":
+                return FlowType.CROSS_PULL_REQUEST;
+            case "full-downstream":
+                return FlowType.FULL_DOWNSTREAM;
+            case "single":
+                return FlowType.SINGLE_PULL_REQUEST;
+            case "branch":
+                return FlowType.BRANCH;
+            default:
+                throw new InvalidInput("Invalid flow-type");
+        }
+    }
+
+    /**
+     * Converts user input to corresponding LoggerLevel enum
+     * @param logLevel value for "logger-level" input
+     * @returns corresponding enum
+     */
+    private getLoggerLevel(logLevel: string): LoggerLevel {
```

Comment:
it is a chance here to refactor this method to
```
    private getLoggerLevel(logLevel: string): LoggerLevel {
        return LoggerLevel[logLevel] as LoggerLevel;
    }
```
you will have to deal with exception and align `logLevel` argument values with LoggerLevel enum values
Up to you

Reply:
Same as above